src/main/java/racingcar/ui/ResultView.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.ui;
+
+import racingcar.service.dto.Cars;
+
+public class ResultView implements Output {
+    private static final String DASH = "-";
+
+    @Override
+    public void output(int number, Cars cars) {
+        System.out.println(String.format("%d 번째 경주", number));
```

Comment:
String.format 제거해주셔도 될 것 같습니다! 

```suggestion
                System.out.printf("%d 번째 경주%n", number);
```